sdk/python/feast/cli.py

```diff
@@ -429,6 +429,30 @@ def materialize_command(start_ts: str, end_ts: str, repo_path: str, views: List[
     )
 
 
+@cli.command("materialize-incremental")
+@click.argument("end_ts")
+@click.argument(
+    "repo_path", type=click.Path(dir_okay=True, exists=True,), default=Path.cwd
```

Comment:
Wait, do we still support `repo_path`? I think we said that we would just allow running Feast commands from the CLI in a repo (not remotely).

Reply:
I haven't been part of that conversation, just copied from materialize command. Should I delete it in both places and replace `FeatureStore(repo_path=repo_path)` with `FeatureStore(repo_path=Path.cwd)`?

Reply:
@tsotnet can you maybe just create a card to remove it? I think it can be a separate PR

Reply:
Done https://tecton.atlassian.net/browse/TEC-4292